main.go

```diff
@@ -58,5 +58,15 @@ func main() {
 	l.Printf("Inventory Service, version %s starting up",
 		CreateVersionString())
 
+	db, err := NewDataStoreMongo(config.Config.GetString(SettingDb))
+	if err != nil {
+		l.Fatal("failed to connect to db")
+	}
+
+	err = db.Migrate(Version, nil)
```

Comment:
I guess only catch would be on interpreting non tagged builds (tags: 1.0.1, 1.2.4 etc) while branches would go into 1.0.x etc